### PR TITLE
Make Python code conform to the Python PEP 8 style guide

### PR DESCRIPTION
**Pull request by practicalswift**
Make Python code conform to the Python PEP 8 style guide.

### CODE REVIEW

File: tests/benchmark.py [hunk 1]
```diff
@@ -1,15 +1,11 @@
...
 import random
-import utils
-
 from concurrent import futures
 from time import time
+import pytest
+from test_lightningd import NodeFactory
 from tqdm import tqdm
-
+import utils
```
Comment (cdecker) [inline tests/benchmark.py]: Mixing module imports with named imports is wrong IMHO, we should separate them.

Comment (dougsland) [reply tests/benchmark.py]: good catch

**Comment by cdecker** [comment]
Just a minor comment about mixing the `import x` and `from y import z` style imports.

ACK 60e9f72

**Comment by practicalswift** [comment]
@cdecker Pushed a new version with imports adjusted – could you take a look? :-)

Yes I excluded `E501` (line too long).

**Comment by practicalswift** [comment]
`pep8` command to use to verify:

```
$ pep8 --ignore=E101,E501,E731,W191 $(git ls-files "*.py")
$
```

**Comment by practicalswift** [comment]
@cdecker Would it be possible to get `pep8` installed on the image that is used on Travis CI?

If so I can add a `pep8` check in the form of `tools/check-python.sh` that would run as part of `make check-source`.

What do you think? :-)

**Comment by dougsland** [comment]
What about creating flake8 Makefile target and call it from check-source target? @practicalswift @cdecker

**Comment by practicalswift** [comment]
@dougsland Yes, that's the idea! See previous comment :-)

**Comment by dougsland** [comment]
I didn't get the check-python.sh. I would suggest:

diff --git a/Makefile b/Makefile
index ce88a881..a2d4e552 100644
--- a/Makefile
+++ b/Makefile
@@ -183,9 +183,14 @@ TEST_GROUP_COUNT=1
 endif
 
 check:
+       $(MAKE) flake8-check
        $(MAKE) installcheck
        $(MAKE) pytest
 
+.PHONY: flake8-check
+flake8-check:
+       flake8
+
 pytest: $(ALL_PROGRAMS)
 ifndef PYTEST
        PYTHONPATH=contrib/pylightning:$$PYTHONPATH DEVELOPER=$(DEVELOPER) python3 tests/test_lightningd.py -f
@@ -230,7 +235,7 @@ check-markdown:
 check-spelling:
        @tools/check-spelling.sh
 
-check-source: check-makefile check-source-bolt check-whitespace check-markdown check-spelling
+check-source: check-makefile check-source-bolt check-whitespace check-markdown check-spelling flake8-check

**Comment by cdecker** [comment]
Sounds good to me, I just want to make sure we aren't creating too many conflicts before merging this one.

**Comment by practicalswift** [comment]
@cdecker It seems like the only open PR touching Python code is #1068? :-)

**Comment by cdecker** [comment]
ACK 870af90

**Pull request merged**